website/docs/r/cis_dns_record.html.markdown

```diff
@@ -8,61 +8,350 @@ description: |-
 
 # ibm_cis_dns_record
 
-Provides a IBM CIS DNS Record resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource. 
+Provides a IBM CIS DNS Record resource. This resource is associated with an IBM Cloud Internet Services instance and a CIS Domain resource.
 
-## Example Usage
+## Example Usage 1 : Create A Record
 
 ```hcl
-# Add a DNS record to the domain
-resource "ibm_cis_dns_record" "example" {
-  cis_id    = ibm_cis.instance.id
-  domain_id = ibm_cis_domain.example.id
-  name      = "terraform"
-  content   = "192.168.0.11"
-  type      = "A"
+resource "ibm_cis_dns_record" "test_dns_a_record" {
+  crn     = var.cis_crn
+  zone_id = var.zone_id
```

Comment:
change attributes crn, zone_id to cis_id and domain_id in all the examples provided in this document
please get domain_id from `ibm_cis_domain` resource or mention domain_id is combination of domain_id or crn_id

Reply:
changes made.